test/Makefile

```diff
@@ -23,9 +23,17 @@ UNSAFE = test_stress
 
 all:
 
-check: $(EXE)
+check: $(EXE) plain-check valgrind-check
```

Comment:
I don't think it's a good idea to require that a `make check` to be executed on Valgrind.
The support for copy and paste was added to Valgrind recently limiting where tests could be executed.
Even if it was added long ago, I don't think manual tests should be forcing tests on top of Valgrind.